### PR TITLE
build: Add script for v2 API env

### PR DESCRIPTION
**Pull request by vsgoulart**

### CODE REVIEW

File: tasklist/Makefile [hunk 1]
```diff
@@ -16,6 +16,11 @@ env-up: $(DATABASE) setup-maven-context
 	$(eval SERVICES := zeebe)
 	$(call run-env)
 
+.PHONY: v2-up
+v2-up: $(DATABASE) setup-maven-context
```
Comment (panagiotisgts) [inline tasklist/Makefile:20]: I believe the V2 API params are already present in the [env](https://github.com/camunda/camunda/blob/c16758258ff00040054dd476152ffe6e99229502/tasklist/config/envs/.env.database.elasticsearch) file for ES/OS. Is there something else missing?

Comment (vsgoulart) [reply tasklist/Makefile:20]: @panagiotisgts You mean I should use these vars instead of hard-coding it?

Comment (panagiotisgts) [reply tasklist/Makefile:20]: They are always being added to the containers/runtime when using the Makefile or the docker-compose

Comment (panagiotisgts) [reply tasklist/Makefile:20]: Ah, I see it's the `elasticsearch` URL reference probably. Can you try adding
`./config/write_or_replace_var.sh "CAMUNDA_DATABASE_URL" "$DATABASE_URL" $MAVEN_CONTEXT`
in the `tasklist/config/env_setup.sh` along the other replacements that happen and try again the `make env-up` goal?

Comment (vsgoulart) [reply tasklist/Makefile]: @panagiotisgts it works, thanks

**Review by panagiotisgts — APPROVED** [review]
Thanks for taking care of the env setup :rocket:

**Pull request merged**